tests/lib/assertions.py

```diff
@@ -46,10 +46,11 @@ def assert_200_response_message(resp, environment):
         hostname = f"{environment}.api.service.nhs.uk"
         prefixes = ["internal-dev", "internal-qa"]
 
-        for p in prefixes:
-            if p in response.get("links").get("self"):
-                hostname = f"{p}-{hostname}"
-                break
+        if environment == 'sandbox':
```

Comment:
Are we sure this shouldn't be getting applied globally? What was the reason for making it sandbox only?

Reply:
Ended up with the test looking for internal-dev-dev in the links ... we don't need to be appending anything when its a non-sandbox environment, just use the environment passed directly in

Reply:
nice ok! just thought I'd check